cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,79 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
+  
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables')
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+  });
+
+  it('should have the ability pagination', () => {
+    cy.get('.-pagination').should('contain', 'Previous')
+      .and('contain', 'Next');
+    cy.contains('.-pageInfo', 'Page').should('exist');
+  });
+
+  it('should have the ability select rows count', () => {
+    cy.get('select[aria-label="rows per page"]').select('25 rows');
+    cy.get('.rt-tr-group', ).should('have.length', 25);
```

Comment:
When you change row count to 5 -> assert that table contain 5 rows. Because now you check value in the dropdown